src/controllers/travelControllers.js

```diff
@@ -46,29 +56,60 @@ export default {
     try {
       const requestsPending = await showManagerPendingAppr(manager);
 
-      const filteredRequests = requestsPending.filter(request => request['user.department.line_manager'] !== null);
+      const filteredRequests = requestsPending.filter(
+        request => request['user.department.line_manager'] !== null
+      );
 
       const requestNumbers = filteredRequests.length;
 
       // eslint-disable-next-line max-len
-      successResponseWithData(res, statusCode.success, message.managerApproval(requestNumbers), filteredRequests);
+      successResponseWithData(
+        res,
+        statusCode.success,
+        message.managerApproval(requestNumbers),
+        filteredRequests
+      );
     } catch (err) {
       errorResponse(res, statusCode.serverError, err);
     }
   },
 
-  getUserTravelStatus: async(req, res) => {
-		const { role, id } = req.userData;
+  getUserTravelStatus: async (req, res) => {
+    const { role, id } = req.userData;
 
-		if (role === 'admin') {
-     return errorResponse(res, statusCode.unauthorized, message.unauthorized);
-		} else {
+    if (role === 'admin') {
+      return errorResponse(res, statusCode.unauthorized, message.unauthorized);
+    } else {
       try {
         const data = await showUsertravelsStatus(id);
-        return successResponseWithData(res, statusCode.success, message.userApproval, data);
+        return successResponseWithData(
+          res,
+          statusCode.success,
+          message.userApproval,
+          data
+        );
       } catch (error) {
+    
         errorResponse(res, statusCode.serverError, error);
       }
     }
-	}
+  }
+};
+
+/* search travels
+ * @param {Object} req - server request
+ * @param {Object} res - server response
+ * @param {Object} next - server response
+ * @returns {Object} - custom response
+ */
+export const searchTravels = async (req, res) => {
+  try {
+    const { body, query } = req;
+    const { page, perPage } = query;
+    const { rows, count } = await searchTravel(body, query);
+    const meta = paginate(page, perPage, count, rows);
+    return res.status(200).json({ success: { requests: rows, meta } });
+  } catch (error) {
+    return res.status(404).json({ error: error });
```

Comment:
@t4christ  Excellent

There is a defined error handler in the utils folder, instead of res.status(404)..... and also a success response too... instead of res.status(200)....